### ISSUE TITLE
Webpack build with 0.6.3 fails

### ISSUE THREAD

**Issue by dzhitomirsky** [sanniassin/react-input-mask#31]
The exceptions is: ReferenceError: navigator is not defined ... at node_modules/react-input-mask/build/InputElement.js:23:18

**Comment by sanniassin** [comment]
@dzhitomirsky  Could you provide your webpack config, version of webpack and build command line?

**Comment by dapi** [comment]
Web pack does not metter.

It fails everytime prerender with node (not in browser)

```
var ua = navigator.userAgent;
              ^
ReferenceError: navigator is not defined
```

Will be better to test browser specification in `componentWillMount`.

**Comment by dapi** [comment]
Here is a test that fails - https://github.com/sanniassin/react-input-mask/pull/34
Here is a fixing - https://github.com/sanniassin/react-input-mask/pull/33

**Comment by sanniassin** [comment]
@dapi Thank you very much! Fixed in 0.6.4
